packages/synthetic-chain/src/cli/run.ts

```diff
@@ -1,11 +1,18 @@
 import { execSync } from 'node:child_process';
+import { realpathSync } from 'node:fs';
 import { ProposalInfo, imageNameForProposal } from './proposals.js';
 
 export const runTestImage = (proposal: ProposalInfo) => {
   console.log(`Running test image for proposal ${proposal.proposalName}`);
   const { name } = imageNameForProposal(proposal, 'test');
+  let slogOpts = '';
+  const { SLOGFILE } = process.env;
+  if (SLOGFILE) {
+    execSync(`touch "$SLOGFILE"`);
+    slogOpts = `-e SLOGFILE -v "$SLOGFILE:${realpathSync(SLOGFILE)}"`;
+  }
```

Comment:
Please DRY out producing `slogOpts`.

Have you considered making it an explicit option instead of an env var? Might help with discovery. Though you're probably the target user so I defer to however you want it to work.

Reply:
> Please DRY out producing `slogOpts`.

Done.

> Have you considered making it an explicit option instead of an env var? Might help with discovery. Though you're probably the target user so I defer to however you want it to work.

I considered it, but there's a lot of layers to penetrate and I think I want this to be dynamically configurable. But it can always be added in the future.